### ISSUE TITLE
`nf-core lint` disagrees with prettier

### ISSUE THREAD

**Issue by Aratz** [nf-core/tools#2606]
### Description of the bug

When running `nf-core lint`, it will rewrite `modules.json` as follows:
```
"installed_by": [
     "modules"
]
```

However, this goes against Prettier, who has strong feelings for
```
"installed_by": ["modules"]
```

#### Steps to reproduce:
1. go to a pipeline repository (e.g. `nf-core/demultiplex`)
2. Run `nf-core lint`, this will change `modules.json`
3. Run `npx prettier --check .`. This will raise a warning about `modules.json`

### System information

- nf-core/tools version 2.11
- prettier v2.8.7

**Comment by maxulysse** [comment]
I think it's a ~~known issue already~~ feature

**Comment by mashehu** [comment]
We had it fixed and I reintroduced this by accident, while making the code run faster. Sorry

**Comment by Aratz** [comment]
No worries :slightly_smiling_face:

**Comment by mashehu** [comment]
fixed in #2609
